viadot/tasks/sql_server.py

```diff
@@ -127,3 +127,41 @@ def run(
             f"Successfully downloaded {nrows} rows and {ncols} columns of data to a DataFrame."
         )
         return df
+
+
+class SQLServerQuery(Task):
+    def __init__(
+        self,
+        config_key: str = None,
+        *args,
+        **kwargs,
+    ):
+        """
+        Task for running query on SQL Server.
+
+        Args:
+            config_key (str, optional): The key inside local config containing the credentials. Defaults to None.
+        """
+        self.config_key = config_key
+
+        super().__init__(name="sql_server_query", *args, **kwargs)
+
+    @defaults_from_attrs("config_key")
+    def run(
+        self,
+        query: str,
+        config_key: str = None,
+    ):
+        """
+        Ran query on SQL Server.
+
+        Args:
+            query (str, required): The query to execute on the SQL Server database.
+            config_key (str, optional): The key inside local config containing the credentials. Defaults to None.
+
+        """
+        sql_server = SQLServer(config_key=config_key)
```

Comment:
Aren't you overriding the default behavior of the source this way? Because the default in the source for `config_key` is "SQL_SERVER", and here by default you'll be passing `config_key=None`.

There should probably be a guard for this in the source (`if config_key is None: config_key="SQL_SERVER"`).

Reply:
I meant in the source class, `SQLServer`. But ok, for now this works too :)